homeworks/natalia.kokovina_NataliKokovina/homework_6/task1.js

```diff
@@ -0,0 +1,31 @@
+function addProtectedKey(firstKey, secondKey, user) {
+    let protectedVal;
```

Comment:
just noticed, you use single variable for both assignments, this is wrong as basically it will store current value for both properties, should be separated

Reply:
fixed